### PR TITLE
Minor borer changes

### PR DESCRIPTION
**Pull request by ShadowDeath6**
### Intent of your Pull Request

Fixes issue #777 by removing borers being able to enter the skeleton race.

Also changes up some of the span class formats to make borer receiving messages prettier. 

Adds a message to the host being infested so they know when to run away.

Also teaches new borers the secret of *talking while in a host.

Also ALSO: fixes the spelling mistake with "telepathicaly" to "telepathically"

### CODE REVIEW

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm [hunk 1]
```diff
@@ -405,8 +406,8 @@ mob/living/carbon/proc/release_control()
 
 		var/mob/living/simple_animal/borer/newborer = new(get_turf(src))
 		newborer.transfer_personality(C)
-		visible_message("<span class='userdanger'>[src] heaves violently, expelling a rush of vomit and a wriggling, sluglike creature!</span>")
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm:408]: I think this one should have been kept userdanger.

**Comment by DavidEdwards** [comment]
> Adds a message to the host being infested so they know when to run away.

Keep it sneaky. Otherwise borers will have no chance.

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm (hunk 1 above)
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm:408]: It's shown to everyone around, so it's kind of annoying to see a giant red message which basically gets the baby borer killed and incriminates the person as well. A smaller message means a tiny bit more stealth which is in my opinion a good thing.

**Comment by ShadowDeath6** [comment]
They have paralyze host for a reason.

**Comment by Time-Green** [comment]
Shadow, they removed that.

**Comment by ShadowDeath6** [comment]
No they didn't.

**Comment by oisin100** [comment]
> They have paralyze host for a reason.

Paralyze shows a message ruining their stealthiness. Borers should be able to sneak into their host as long as they dont use paralyze. Keep infecting silent.

**Comment by ShadowDeath6** [comment]
The point of that is to keep players from bitchmoaning about a borer getting into their head while showing literally no indication other than it appearing for about a second and a half before teleporting into their head. It's only fair to give players a heads-up that a borer will infect them. 
Besides, if you time it right, paralyze will give enough time to infect them.

File: code/modules/mob/living/simple_animal/borer/borer_powers.dm (hunk 1 above)
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer_powers.dm:408]: > Adds a message to the host being infested so they know when to run away.

Ye, Real stealthy.

If you want your borer to survive reproduction go to some secluded place and do it. Borers are a "threat" to anyone nearby them thus the userdanger message. Its already easy pez to reproduce, dont make it any easier.

**Comment by AsV9** [comment]
Unless @ShadowDeath6 can cough up some support for this, it will be closed in 24hrs.
I'm personally opposed to most of these changes.

**Comment by ShadowDeath6** [comment]
What are you opposed to, asv? I can remove the message from infesting but the span classes for old borer things were really ugly and wizard still gets bugged  in the very unlikely case it makes a borer.

The message encourages borers to actually play smart and try to stun a host instead of relying on complete invisibility in text logs and then making someone salty because there's literally no warning other than the HUD that pops up. Bay did it, it worked fine.

**Comment by AsV9** [comment]
We're not bay. I'd like borers to be stealthy and be able to infiltrate hosts without a giant userdanger message popping up encouraging them to bee-line to the chemist to make sugar.

**Comment by ShadowDeath6** [comment]
It's not like they don't get an HUD over their head and a borer talking to them. It would give people a chance to actually run away when a borer tried to infect them, which is only fair. I can make it smaller if you think it'll be that much of a problem.

**Comment by DavidEdwards** [comment]
I'm honestly not sure we should have a HUD added for the borer host either, unless the borer allows it.

That is something that has bugged me a little bit.

**Comment by AsV9** [comment]
I'm kinda with Kn0ss0s on that issue aswell. The borer should have the HUD, not the host, forcing them to communicate if they want to share that information.

**Comment by ShadowDeath6** [comment]
I think that was originally indended. Does that not happen?

**Comment by ShadowDeath6** [comment]
tl;dr people want borer messages when I ask but nobody stands up for me when I do if

I'll comment it out in case somebody wants it later.

**Comment by ShadowDeath6** [comment]
I need to sleep, I hope I've appeased you. ;_;
